src/webview/wkwebview/mod.rs

```diff
@@ -287,6 +295,20 @@ impl InnerWebView {
                 }
               }
             }
+
+            // Key event chain is consumed by window and cannot pass to menu.
+            // So we pass the event to menu if we have one
+            extern "C" fn key_down(_: &mut Object, _: Sel, event: id) {
+              unsafe {
+                let app = cocoa::appkit::NSApp();
+                let menu: id = msg_send![app, mainMenu];
+                if !menu.is_null() {
+                  let () = msg_send![menu, performKeyEquivalent: event];
+                }
+              }
+            }
+            // We need to declare this method to get Command+ key equivalent.
+            extern "C" fn do_command_by_selector(_: &Object, _: Sel, _: Sel) {}
```

Comment:
can this fix be upstream to `winit`? seems like a valid use-case for any winit app.

Reply:
winit can call `performKeyEquivalent` in `WinitWindow`'s `sendEvent` but I'm not sure if winit team willing to take this workaround. ket event chain in macOS is a mess tbh.

As for workaround for users of winit, I think we can add a NSView as subview and add similar code like above. Or just replace `winitWindow` or default NSView if they don't care any window events sent from winit.

Reply:
winit ships with a default menu so this workaround is valid for their use-case too but we won't know for sure until we open a PR.